ReactAndroid/src/main/java/com/facebook/react/devsupport/MultipartStreamReader.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react.devsupport;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import okio.Buffer;
+import okio.BufferedSource;
+import okio.ByteString;
+
+/**
+ * Utility class to parse the body of a response of type multipart/mixed.
+ */
+public class MultipartStreamReader {
```

Comment:
Is there some utility in OkHttp that could help with this, e.g. https://square.github.io/okhttp/3.x/okhttp/okhttp3/MultipartBody.html?